spark/common/pom.xml

```diff
@@ -253,4 +253,20 @@
             </build>
         </profile>
     </profiles>
+
+    <build>
```

Comment:
We can put this configuration section in `spark/pom.xml` to avoid configuring each submodule separately.

Reply:
Thanks for your suggestion